geomesa-accumulo/geomesa-accumulo-spark/src/main/scala/org/locationtech/geomesa/spark/accumulo/AccumuloSpatialRDDProvider.scala

```diff
@@ -94,34 +59,9 @@ class AccumuloSpatialRDDProvider extends SpatialRDDProvider with LazyLogging {
 
         // From sc.newAPIHadoopRDD
         // Add necessary security credentials to the JobConf. Required to access secure HDFS.
-        val jconf = new JobConf(conf)
+        val jconf = job.getConfiguration.asInstanceOf[JobConf]
         SparkHadoopUtil.get.addCredentials(jconf)
 
-        // Get username from params
-        val username = AccumuloDataStoreParams.UserParam.lookup(paramsAsJava)
```

Comment:
Where does/did this logic / configuration / etc end up?

Reply:
this is all handled by `GeoMesaAccumuloInputFormat.configure` now. we had a lot of duplicate code between these two classes, i think b/c the main input format requires a Job instance and we didn't have one here.

Reply:
Great!  Is there a similar refactoring that should happen on the HBase side?  Or is that code happy?

Reply:
hbase is good already: https://github.com/locationtech/geomesa/blob/master/geomesa-hbase/geomesa-hbase-spark/src/main/scala/org/locationtech/geomesa/spark/hbase/HBaseSpatialRDDProvider.scala#L53